cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -40,15 +41,18 @@ def _delete_vm():
 
 def verify_vm_power_state(vm_obj, state, action=None):
     vm = vm_obj.get_vm_via_rest()
-    if vm_obj.provider.one_of(GCEProvider, EC2Provider):
-        num_sec = 1600  # extra time for slow providers
+    if vm_obj.provider.one_of(GCEProvider, EC2Provider, SCVMMProvider):
```

Comment:
There's also `vm_obj.provider.one_of(GCEProvider, EC2Provider)` in other place, so I would rather leave it explicit as it is since there are different combinations in different places.